docs/UserGuide.md

```diff
@@ -352,6 +352,15 @@ If your changes to the data file makes its format invalid, DonnaFin will discard
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous DonnaFin home folder.
 
+**Q**: I have multiple clients with the same name, but your application won't let me add them! What do I do?
+**A**: We realise some names are common, but we decided to prioritise your ability to instantly recognise a client over letting you keep the accuracy of the name. For that reason, we suggest adding other identifying nicknames or words in the name e.g. "John Walker (Bartender)" and "John Walker (Johnny)".
```

Comment:
tbh im not very convinced by this since there may come a time where there are too many clients to keep track of their nicknames. but at the same time im not sure what else to do...

Reply:
Same but no choice right?